libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -33,6 +33,8 @@
 #include <libsolutil/Whiskers.h>
 #include <libsolutil/StackTooDeepString.h>
 
+#include <libsolidity/codegen/GenericStorageItem.cpp>
+
```

Comment:
Can't you do some external template instantiation instead? That way we could also keep this in ``LValue.cpp`` and actually in-place modify ``StorageItem`` there, which would make the diff much easier to work through.